control-plane/connect-inject/controllers/pod/pod_controller_ent_test.go

```diff
@@ -330,7 +330,7 @@ func TestReconcileDeletePodWithMirrorNamespaces(t *testing.T) {
 			existingWorkload:           createWorkload(),
 			existingHealthStatus:       createPassingHealthStatus(),
 			existingProxyConfiguration: createProxyConfiguration(testPodName, pbmesh.ProxyMode_PROXY_MODE_DEFAULT),
-			existingUpstreams:          createUpstreams(),
```

Comment:
should be `existingDestinations:`. Just double check for other instances of upstream in this and other files